src/library/dlgtrackinfo.cpp

```diff
@@ -21,11 +24,12 @@ const int kMinBpm = 30;
 // Maximum allowed interval between beats (calculated from kMinBpm).
 const mixxx::Duration kMaxInterval = mixxx::Duration::fromMillis(1000.0 * (60.0 / kMinBpm));
 
-DlgTrackInfo::DlgTrackInfo(QWidget* parent)
-            : QDialog(parent),
-              m_pTapFilter(new TapFilter(this, kFilterLength, kMaxInterval)),
-              m_dLastTapedBpm(-1.),
-              m_pWCoverArtLabel(new WCoverArtLabel(this)) {
+DlgTrackInfo::DlgTrackInfo(QWidget* parent, UserSettingsPointer pConfig)
```

Comment:
I vaguely remeber that the parent parameter should always be last, right?

Reply:
I guess there is no consistent scheme. Afterwards I noticed that in many library components we also use it as the first parameter. If it is a generic `QObject` with default value `nullptr ` then it should be the last parameter, otherwise choose whatever you like.

Reply:
@daschuer does it need to be the last argument for `parented_ptr`?

Reply:
I'll just swap the just in case.

Reply:
No, the order does not matter.